spec/model/post_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
```

Comment:
- [ ] Again here, It is a requirement that Model specs has **Validation** tests, these are the validations in the `post` model. Kindly implement this.
-For example:

```
describe 'validations' do
  it { should validate_presence_of(:title) }
end
```

- [ ] Kindly ensure to implement tests for all the validations in the `post` model,